src/supernode/src/test/java/com/alibaba/dragonfly/supernode/repository/TaskRepositoryTest.java

```
@@ -0,0 +1,148 @@
+package com.alibaba.dragonfly.supernode.repository;
+
+import com.alibaba.dragonfly.supernode.common.domain.Task;
+import com.alibaba.dragonfly.supernode.common.enumeration.CdnStatus;
+import com.alibaba.dragonfly.supernode.repository.TaskRepository;
+import com.diffblue.deeptestutils.Reflector;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+public class TaskRepositoryTest {
+
+  @Rule public ExpectedException thrown = ExpectedException.none();
+
+  /* testedClasses: TaskRepository */
+
+  /*
+   * Test generated by Diffblue Deeptest.
+   * This test case covers:
+   *  - conditional line 90 branch to line 90
+   *  - conditional line 90 branch to line 91
+   */
+
+  @Test
+  public void computePieceSizeInputNegativeOutputPositive()
```

👍  Bug? - negative file length shouldn't be silently accepted.